examples/jsm/nodes/display/ViewportNode.js

```diff
@@ -95,7 +95,7 @@ class ViewportNode extends Node {
 
 			let coord = builder.getFragCoord();
 
-			if ( builder.isFlipY() ) {
+			if ( builder.isFlipY() && ! builder.renderer.backend.isWebGLBackend ) {
```

Comment:
How about this instead:
```js
if ( builder.isFlipY() && builder.renderer.backend.isWebGPUBackend ) {
```
Isn't that a bit more readable?

Reply:
Agreed! Also this fails on Puppeteer, will have to investigate on why before merging but it might be related to the fact WebGPU is available but cannot start an adapter on there.

It would also be nice to be able to force the WebGL Backend even if WebGPU is available on the web browser.

Reply:
The idea of having `isFlipY()` is that we don't need to test which backend is the target. 

```js
isFlipY() === true // always be WebGL Backend
isFlipY() === false // always be WebGPU Backend
```

Reply:
Maybe this issue is related to a `RenderTarget`?

Reply:
If I understand correctly this mimic the behavior of glFragCoord as it's unavailable on WebGPU so basically this code isn't necessary in WebGL and we can simply return the string instead. Seems to fix WebGL.

Not sure what's the issue with Pupeeteer, maybe it breaks all the materials I will have to investigate later I'm out of keyboard for a few days 😊

Reply:
`viewportCoordinate` is `vec2( gl_FragCoord.x, resolution.y - gl_FragCoord.y )` for WebGL with y-flipped, or `fragCoord` for WebGPU.

Reply:
I found a bug in the build of node, may this issue could it? https://github.com/mrdoob/three.js/pull/27566